docs/source/api_reference/forecasting.rst

```diff
@@ -211,6 +211,18 @@ All "ARIMA" and "Auto-ARIMA" models below include SARIMAX capability.
 
     ARIMA
 
+.. currentmodule:: sktime.forecasting.statsforecast
```

Comment:
I think this is a copy-paste accident - I moved the estimators around in a previous PR. MSTL belongs with STL rather than ARIMA.